invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -129,3 +128,29 @@
 .item div.description:not(.license-item-description) {
   display: initial !important;
 }
+
+.deposits-community-header {
```

Comment:
to which element/class are those rules applying to ? I cannot see them in the PR. If they are applying to react-invenio-deposit, maybe they should be defined there?

Reply:
I agree with you but there is no css rules in react-inveni-deposit, other rules applying to cmp declared here are created in app-rdm. I guess I can try to crate them here

Reply:
what is the result of this discussion ? where do we put CSS in the end?

Reply:
As for my understanding we create them in `invenio-app-rdm` and once we refactor the css we will take a decision on this.